docs/pages/Guides/Customising-prompts.md

```diff
@@ -1,4 +1,28 @@
-## To customize a main prompt navigate to `/application/prompt/combine_prompt.txt`
+# Customizing the Main Prompt
 
-You can try editing it to see how the model responses.
+To customize the main prompt for DocsGPT, follow these steps:
+
+1. Navigate to `/application/prompt/combine_prompt.txt`.
+
+2. Edit the `combine_prompt.txt` file to modify the prompt text. You can experiment with different phrasings and structures to see how the model responds.
+
+## Example Prompt Modification
+
+**Original Prompt:**
+```markdown
+You are a DocsGPT, friendly and helpful AI assistant by Arc53 that provides help with documents. You give thorough answers with code examples if possible.
+Use the following pieces of context to help answer the users question. If its not relevant to the question, provide friendly responses.
+You have access to chat history, and can use it to help answer the question.
+When using code examples, use the following format:
+
+(code)
+{summaries}
+
+Thank you
+
+
```

Comment:
Can we add one more example
from here https://docs.docsgpt.co.uk/Guides/My-AI-answers-questions-using-external-knowledge

And maybe one check out the https://docs.anthropic.com/claude/docs/let-claude-say-i-dont-know
and https://docs.anthropic.com/claude/docs/ask-claude-to-think-step-by-step
https://docs.anthropic.com/claude/docs/break-tasks-into-subtasks

As a source of inspiration for one or two prompts with some comments next to them